packages/plugin-ext/src/main/browser/view/plugin-tree-view-node-label-provider.ts

```diff
@@ -43,12 +43,14 @@ export class PluginTreeViewNodeLabelProvider implements LabelProviderContributio
         if (node.icon) {
             return node.icon;
         }
-        if (node.themeIconId) {
-            if (node.themeIconId === 'file' || node.themeIconId === 'folder') {
+        if (node.themeIcon) {
+            if (node.themeIcon.id === 'file' || node.themeIcon.id === 'folder') {
```

Comment:
I just wanted to highlight that this only allows us a subset of icons to be used which is different than the API actually allows. VS Code allows all icons listed here (https://code.visualstudio.com/api/references/icons-in-labels#icon-listing) which is documented in `id`:

```ts
/**
 * The id of the icon. The available icons are listed in https://code.visualstudio.com/api/references/icons-in-labels#icon-listing.
 */
readonly id: string;
```

With some modifications to the extension provided in VS Code I can see (while in Theia it does not work):


![image](https://user-images.githubusercontent.com/40359487/171661936-f7360585-55f1-4cbc-b72c-3abf9c9aa83a.png)